.ci/scripts/wpe-testbed.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import re
+import subprocess
+import sys
+
+import xml.etree.ElementTree as ET
+
+TESTS = ('unbounded', 'unbounded_and_no_animate')
+
+def parse_args(argument_list):
+    parser = argparse.ArgumentParser(
+        description="Run WPE testbed.",
+    )
+    parser.add_argument('--new-baseline', action='store_true', dest='new_baseline', help='Save generated results as new baseline')
+    parser.add_argument('--report', action='store_true', dest='report', help='Return actual and expected results')
+    parser.add_argument('--filter', dest='filter', choices=TESTS, help='Filter by test name')
+    return parser.parse_args(argument_list)
+
+# Utility class to compose run wpe-testbed.
+class WPETestbed:
+
+    WAYLAND_DISPLAY="/run/wayland-0"
+    program = "wpe-testbed-wayland"
+
+    def __init__(self, **kwargs):
+        # Set default values.
+        if "tiles" not in kwargs:
+            self.tiles = 1
+        if not "opaque" in kwargs:
+            self.opaque = True
+        if not "fences" in kwargs:
+            self.fences = True
+        if not "title_update_method" in kwargs:
+            self.title_update_method = "mmap"
+        if not "dmabuf_tiles" in kwargs:
+            self.dmabug_tiles = True
+        if not "frames" in kwargs:
+            self.frames = 2000
+        if not "drm_node_ipu" in kwargs:
+            self.drm_node_ipu = "/dev/dri/card0"
```

Comment:
notice that this value is different in the `wandboard-mesa` (i.MX6 board) so this depends on the hardware where we want to execute the command. In the `wandboard-mesa`'s case is `--drm-node-ipu /dev/dri/card1`